### DISCUSSION TITLE
Phantom issue subscriptions

### DISCUSSION THREAD

**Discussion by fatcerberus** [community/community#17565] [Other Feature Feedback, Questions, & Ideas]
I just cleared out my issue subscriptions and noticed that GH thinks there's still 90 issues I'm subscribed to despite the list being empty:

![image](https://user-images.githubusercontent.com/4097841/171282443-e881f723-c3aa-4e46-acc4-162f9468a611.png)

Not a huge issue, but kind of annoying (OCD and all).  Any way to get rid of the badge?  I've tried everything and can't find a way to clear it.

**Comment by Arvind-Srinivasan** [comment]
I too have this issue
<img width="1023" alt="Screen Shot 2022-07-12 at 21 38 18" src="https://user-images.githubusercontent.com/15947763/178629897-10f52fb6-5333-4404-9810-3b41a3012010.png">
